client/hwlib/debian/vendor-rust.sh

```diff
@@ -7,7 +8,16 @@ if [ ! $(which cargo-vendor-filterer) ]; then
     exit 3
 fi
 
+cargo vendor-filterer "${CARGO_VENDOR_DIR:-vendor}"
+
 # Some crates are shipped with .a files, which get removed by the helpers during the package build as a safety measure.
 # This results in cargo failing to compile, since the files (which are listed in the checksums) are not there anymore.
-# For those crates, we need to replace their checksum with a more general one that only lists the crate checksum, instead of each file.
-cargo vendor-filterer "${CARGO_VENDOR_DIR:-vendor}"
+# For those crates, we need to remove their checksum
+find "${CARGO_VENDOR_DIR:-vendor}" -name ".cargo-checksum.json" -exec bash -c '
```

Comment:
This is somewhat hard to read. Peharps something more along the lines:
```sh
for json_file in $(find ...); do
   ...
done
```

Make sure to shellcheck the logic.

Reply:
I've tried to simplify the logic with your suggestion